packages/react-i18n/src/i18n.ts

```diff
@@ -548,3 +560,15 @@ function dateTimeFormatter(
 ) {
   return new Intl.DateTimeFormat(locale, options);
 }
+
+function getSeparators(str: string) {
+  let thousandSeparator;
+  let decimalSeparator;
+  for (const char of str) {
+    if (isNaN(parseInt(char, 10))) {
```

Comment:
What is the reason for using `parseInt()` here?

```suggestion
    if (isNaN(char)) {
```

`isNaN(char)` will return the same value as `isNaN(parseInt(char, 10))` and will be faster. 
Maybe I'm missing something?

Reply:
Some languages use a space for the thousand operator (e.g. `fr-FR`). `isNaN(' ')` is `false` although it should be true so I'm using `parseInt(' ', 10)` to first convert the string into a number or `NaN`

Reply:
TIL that for JavaScript, `' '` is a number. Nice catch!